File: esports-backend/src/main/java/com/esports/dto/KycStatusUpdateDTO.java
package com.esports.dto;

public class KycStatusUpdateDTO {
    private String kycStatus;

    public KycStatusUpdateDTO() {}

    public KycStatusUpdateDTO(String kycStatus) {
        this.kycStatus = kycStatus;
    }

    public String getKycStatus() {
        return kycStatus;
    }

    public void setKycStatus(String kycStatus) {
        this.kycStatus = kycStatus;
    }
}

File: Upkart1/DTO/KycStatusUpdateDTO.cs
namespace Upkart1.DTO
{
    public class KycStatusUpdateDTO
    {
        public string KycStatus { get; set; }

        public KycStatusUpdateDTO() { }

        public KycStatusUpdateDTO(string kycStatus)
        {
            KycStatus = kycStatus;
        }
    }
}
